addons/mail/static/src/owl/components/composer/composer.scss

```diff
@@ -149,15 +173,43 @@
                 }
             }
         }
+    }
+
+    &.o-is-compact {
+        .o_Composer_buttonSend {
+            &:disabled {
+                border-left: none;
+            }
+        }
+    }
+
+    &:not(.o-is-compact) {
+
+        .o_Composer_coreMain {
+            background: white;
+            border-radius: 5px;
+            border: 1px solid lighten(gray('400'), 5%);
+        }
+
+        .o_Composer_textInput {
+            border: 0;
+        }
 
+        .o_Composer_buttons {
+            border: 0;
+
+            .o_Composer_toolButtons{
+                border-bottom: 0;
+                border-radius: initial;
+            }
+        }
     }
 }
 
 .o_Composer_button {
 
     &:focus {
-        box-shadow: none;
-        background-color: gray('200');
+        box-shadow: 0 0 0 0.2rem rgba(38, 174, 172, 0.5);
```

Comment:
definitely should not be hard-coded colour. Looks like odoo secondary colour?

Reply:
Yep, this rule is useless by the way